### ISSUE TITLE
Request to output warning and error data so flow can handle exceptions

### ISSUE THREAD

**Issue by davidplatt** [tilialabs/tilia-phoenix-connect#23]

**Comment by tiliadavid** [comment]
@pkleinheider We are working on a completely new version of the app that behaves differently that would make error handling a lot more robust - this is still in the works though so this issue has been put on ice for the time being. If you make a change to implement this we'd be happy to review it via a pull request!

**Comment by pkleinheider** [comment]
Id just add two outgoing connections, one for the job (with the option of the error log as dataset) and one for a log file with an output connection of type Data: Processing Error 
and only of this output connection is not defined, do a job.fail.

**Comment by pkleinheider** [comment]
I guess you rewrite the configurator in TypeScript, correct?

**Comment by tiliadavid** [comment]
@pkleinheider That is correct - that was the main impetus for the change as well, to move to TypeScript.
